### ISSUE TITLE
Logged Zenodo link points to an older version

### ISSUE THREAD

**Issue by mielliott** [bio-guoda/preston#24]
Running `preston update` on version `0.1.4` gave following log:
```shell
https://doi.org/10.5281/zenodo.1410543  http://purl.org/dc/terms/bibliographicCitation  Jorrit Poelen, Icaro Alzuru, & Michael Elliott. 2019. Preston: a biodiversity dataset tracker (Version 0.1.4) [Software]. Zenodo. http://doi.org/10.5281/zenodo.3334454
```
The first link points to the Preston itself, not specifically `0.1.4` (https://doi.org/10.5281/zenodo.1410543), but the second points to version `0.1.2` (http://doi.org/10.5281/zenodo.3334454).

**Comment by jhpoelen** [comment]
Oops! Nice catch. Would you like to fix this one?

**Comment by mielliott** [comment]
@jhpoelen gladly!

**Comment by mielliott** [comment]
Fixed in f65fa11d4a15bed4041a9ea12599595ba37bffca. I just changed the second link to be the same as the first link. It would be tricky to make the second link point to the exact version of Preston used since the Zenodo ID for that version is generated after build, but that ID would need to be stamped into the .jar.

**Comment by jhpoelen** [comment]
@mielliott nice! Note that you can automatically link a commit to an issue if you use ```#1235``` where 1234 is the issue number. When referencing links in other repositories, include the whole link, like ```https://github.com/bio-guoda/preston/issues/24```
